SYCL/ESIMD/api/functional/ctors/operator_assignment_copy_core.cpp

```diff
@@ -0,0 +1,118 @@
+//==------- operator_assignment_copy_core.cpp  - DPC++ ESIMD on-device test
+//          ----------------------------------------------------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu, level_zero
+// XREQUIRES: gpu
+// TODO gpu and level_zero in REQUIRES due to only this platforms supported yet.
+// The current "REQUIRES" should be replaced with "gpu" only as mentioned in
+// "XREQUIRES".
+// UNSUPPORTED: cuda, hip
+// RUN: %clangxx -fsycl %s -fsycl-device-code-split=per_kernel -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+//
+// Test for simd copy assignment operator.
+// The test creates source simd instance with reference data and invokes copy
+// assignment operator. It is expected that simd object that called copy
+// assignment operator will stor a bitwise same data as in the source simd
+// object.
```

Comment:
```suggestion
// assignment operator from source to destination simd instance. It is expected
// for destination simd instance to store a bitwise same data as the reference one
```

Reply:
I've done it in this commit: c08d8bcd8cfb2adfcd4af2321500e8672a2b471c